README.md

```diff
@@ -7,19 +20,25 @@
 
 ## Without Docker
 
+The following commands will run for all the workspaces. If you want to run a command for one workspace add the -w command. For example, to add a package to the dataset-browser:
+
+    npm install myPackage --save-exact -w dataset-browser
+
 ### Install packages
 
-    npm install --no-progress
+Install all the packages, including the apps and packages
```

Comment:
This sentence contains the word 'packages' two times - reformulate a little bit? Or just remove ', including the apps and packages'?